frontend/src/components/Rooms/WaitingRoom.tsx

```diff
@@ -0,0 +1,141 @@
+import { Component, createSignal, For, createEffect } from 'solid-js';
+import { useNavigate } from '@solidjs/router';
+import axios from 'axios';
+import { writeClipboard } from '@solid-primitives/clipboard';
+
+const WaitingRoom: Component = () => {
+    const navigate = useNavigate();
+    const [players, setPlayers] = createSignal([]);
+    const [currentPlayerCount, setCurrentPlayerCount] = createSignal(0);
+    const [maxPlayerCount, setMaxPlayerCount] = createSignal(4);
+    const [roomCode, setRoomCode] = createSignal('');
+    const [deckCount, setDeckCount] = createSignal(1);
+    const [cardsPerPlayer, setCardsPerPlayer] = createSignal(0);
+    const [jokerCount, setJokerCount] = createSignal(0);
+
+    createEffect(() => {
```

Comment:
this effect runs anytime any of the dependent signal values are set. Does not seem intended

Reply:
not intended, fixed now